src/styles/product-card.scss

```
@@ -0,0 +1,99 @@
+.product-card {
+  box-sizing: border-box;
+  padding: 0 16px;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+
+  justify-content: center;
+
+  background: $bg-color;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__item {
+    display: flex;
+    justify-content: center;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-top: 32px;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    color: $text-color;
+
+    margin: 4px 0 0;
+  }
+
+  &__name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+
+    margin: 40px 0 0;
+  }
+
+  &__price {
+    display: inline-flex;
+
+    margin-top: 12px;
+    margin-bottom: 0;
+  }
+
+  &__text {
+    font-size: 12px;
+    line-height: 18px;
+    color: $text-color;
+
+    align-items: flex-start;
+    text-align: left;
+  }
+
+  &__cost {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+
+    margin-top: 12px;
+    margin-left: 86px;
+  }
+
+  &__button {
+    margin-bottom: 16px;
+  }
+
+  &__link {
+    justify-content: center;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    text-decoration: none;
+    color: $bg-color;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-transform: uppercase;
+
+    background-color: $button-main-color;
+    width: 166px;
+    height: 40px;
+    border-radius: 5px;
+
+    margin: 0;
+
+    &:hover {
+      box-sizing: border-box;
+      color: $button-main-color;
+      background-color: $bg-color;
+      border: 1px solid $button-main-color;
+      border-radius: 5px;
+      width: 166px;
```

No need repeat width and border-radius
```suggestion
      width: 166px;
```
```suggestion
```